app/decorators/rejection_decorator.rb

```diff
@@ -12,11 +12,9 @@ def allowance_renews_on
     @allowance_renews_on ||=
       begin
         if object.allowance_renews_on
-          AccessibleDate.new(
-            date_to_accessible_date(object.allowance_renews_on)
-          )
+          AccessibleDate.new(date_to_accessible_date(object.allowance_renews_on))
         else
-          AccessibleDate.new
+          AccessibleDate.from_multi_parameters(allowance_renews_on_before_type_cast)
```

Comment:
it's because before `Rejection#allowance_renews_on=` would retain the assigned value wether it was valid or not but with the newer active record the attribute/instance variable gets wiped out by the newer `ActiveRecord`